### DISCUSSION TITLE
Text color for success text and accessibility

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1344] [Q&A] [answered]
Hello there, 
In the Paste website shows text color for success text is failed a11y. So if I want to use it for a design, what solution(color) would you recommend? 
<img width="503" alt="Screen Shot 2021-04-07 at 11 45 25 AM" src="https://user-images.githubusercontent.com/67388401/113918180-f5396500-9796-11eb-8c67-6a01cde65bdb.png">

**Comment by serifluous** [comment, marked as answer]
Hey @sang-yeo ! Yep, that's more of an error on our part with our token naming. `$color-text-success` is currently meant only for icon colors. We'll be updating our token names eventually so it's clearer.

For now, we recommend using the default text color `$color-text` and if needed, pairing it with a success icon in the success color. If that doesn't work for your case, let us know!

**Reply by sang-yeo** [reply]
Thanks for clarifying!

**Comment by ishah-twilio** [comment]
To followup to Sang's question.

Can you please help me understand what icon we should use for Service status that could be Disabled / Enabled? Should we use Success icon and Error icons for Disabled / Enabled status?

Attached is the screen for more context -- Status of service enabled for the Fleet feature.

<img width="888" alt="Screen Shot 2021-04-07 at 1 07 39 PM" src="https://user-images.githubusercontent.com/69016180/113928283-6d0d8c80-97a3-11eb-85e9-f7c9f809dda2.png">

**Reply by serifluous** [reply]
For this screen, are you trying to call out both the disabled and enabled states? Does the customer need to take action if data and/or SIM is enabled?

In a case like this, where you're showing lots of different statuses (including the warning at the top of the modal), I'd recommend using an icon or different text color **only** when you need to call attention to the information that the customer absolutely has to pay attention to or take action on. Limiting icon usage to just one of the states could make it a little more straightforward for you to decide what icon you should use.

If you do need icons for both states, though, or you're still not confident on what icon to use, let us know and we can brainstorm more options with you.
